GENICAM/iocBoot/iocGENICAM-IOC-01/devices/pearlcam_dbpfs.cmd

```diff
@@ -34,3 +34,8 @@ dbpf "$(PREFIX)CAM1:GC_BalRatioSelector", "0"
 
 dbpf "$(PREFIX)CAM1:GC_ExposureMode", "0"
 dbpf "$(PREFIX)CAM1:GC_ExposureAuto", "2"
+
+dbpf "$(PREFIX)CAM1:GC_DevLinThrLimit", "2.63726e+06"
```

Comment:
This is a bit of a magic number... can this please have a comment explaining how we got to this number?

Reply:
I just stole it from what the scientists had in spinview. I think it's in bytes, so it should be ~2.3MB/S?